src/Bundle/Tests/Routing/CrudRoutesAttributesLoaderTest.php

```diff
@@ -24,7 +24,7 @@ public function it_generates_crud_routes_from_resource_alias(): void
     {
         self::bootKernel(['environment' => 'test_with_attributes']);
 
-        $container = static::$container;
+        $container = \method_exists($this, 'getContainer') ? static::getContainer() : static::$container;
```

Comment:
for which version do we need this check?

Reply:
done